apps/api/src/domain/models/audio-visual/shared/commands/transcripts/add-participant-to-transcript/participant-added-to-transcript.event-handler.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { ArangoConnectionProvider } from '../../../../../../../persistence/database/arango-connection.provider';
+import { ArangoDatabaseProvider } from '../../../../../../../persistence/database/database.provider';
+import { PersistenceModule } from '../../../../../../../persistence/persistence.module';
+import generateDatabaseNameForTestSuite from '../../../../../../../persistence/repositories/__tests__/generateDatabaseNameForTestSuite';
+import { buildTestInstance } from '../../../../../../../test-data/utilities';
+import { EventSourcedAudioItemViewModel } from '../../../../audio-item/queries';
+import { TranscriptParticipant } from '../../../entities/transcript-participant';
+import { Transcript } from '../../../entities/transcript.entity';
+import { ParticipantAddedToTranscript } from './participant-added-to-transcript.event';
+import { ParticipantAddedToTranscriptEventHandler } from './participant-added-to-transcript.event-handler';
+
+const audioItemId = buildDummyUuid(50);
+
+const targetAudioItem = buildTestInstance(EventSourcedAudioItemViewModel, {
+    id: audioItemId,
+    transcript: Transcript.buildEmpty(),
+});
+
+const participantAdded = buildTestInstance(ParticipantAddedToTranscript, {
+    payload: {
+        aggregateCompositeIdentifier: { id: targetAudioItem.id, type: AggregateType.audioItem },
+    },
+});
+
+describe('ParticipantAddedToTranscriptEventHandler', () => {
+    let databaseProvider: ArangoDatabaseProvider;
+
+    let app: INestApplication;
+
+    let testRepositoryProvider: IQueryRepositoryProvider;
+
+    let participantAddedToTranscriptEventHandler: ParticipantAddedToTranscriptEventHandler;
+
+    beforeAll(async () => {
+        const moduleRef = await Test.createTestingModule({
+            imports: [PersistenceModule.forRootAsync()],
+        })
+            .overrideProvider(ConfigService)
+            .useValue(
+                buildMockConfigService(
+                    {
+                        ARANGO_DB_NAME: generateDatabaseNameForTestSuite(),
+                    },
+                    buildConfigFilePath(Environment.test)
+                )
+            )
+            .compile();
+
+        await moduleRef.init();
+
+        app = moduleRef.createNestApplication();
+
+        const connectionProvider = app.get(ArangoConnectionProvider);
+
+        databaseProvider = new ArangoDatabaseProvider(connectionProvider);
+
+        testRepositoryProvider = app.get(QUERY_REPOSITORY_PROVIDER_TOKEN);
+
+        participantAddedToTranscriptEventHandler = new ParticipantAddedToTranscriptEventHandler(
+            // @ts-expect-error TODO find a better way to sidestep this type issue
+            testRepositoryProvider
+        );
+    });
+
+    afterAll(async () => {
+        databaseProvider.close();
+    });
+
+    beforeEach(async () => {
+        await databaseProvider.clearViews();
+
+        /**
+         * We attempted to use "handle" on a creation event for the test
```

Comment:
This comment definitely doesn't apply to our test. Copy and paste error? Remove this.

Reply:
I went ahead and did this for you as I rebased onto my fix branch for the command handlers.